modules/ting_search_ortograf/includes/ting_search_ortograf.admin.inc

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * @file
+ * Admin settings for bibliotek.dk autocomplete.
+ */
+
+/**
+ * Ting Search Ortograf admin form
+ */
+function ting_search_ortograf_admin() {
+  $default_settings = ting_search_ortograf_settings();
+
+  // Attach the CSS to the form.
+  $path = drupal_get_path('module', 'ting_search_ortograf');
+  $form['#attached'] = array(
+    'css' => array(
+      'type' => 'file',
+      'data' => $path . '/css/ting_search_ortograf_admin_form.css',
+    ),
+  );
+
+  $form['ting_search_ortograf_webservice'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Autocomplete URL', array(), array('context' => 'ting_search_ortograf')),
+    '#tree' => FALSE,
+    '#collapsible' => TRUE,
+  ); 
+
+  $form['ting_search_ortograf_webservice']['ting_search_ortograf_suggestion_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Autocomplete URL', array(), array('context' => 'ting_search_ortograf')),
+    '#size' => 73,
```

Comment:
Why set this very specific size?

If we want to control the layout then please use the style sheet.